kernel/kernel-config-4.9.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86_64 4.9.60 Kernel Configuration
+# Linux/x86 4.9.60 Kernel Configuration
```

Comment:
This looks a strange I dont thing should change, can you double check you wrong changed this. It is generated message but just in case.

Reply:
This is what the result is in   .config is when using OSBuilder to build the kernel "make kernel".